chainer/training/extensions/log_report.py

```diff
@@ -139,9 +174,13 @@ def serialize(self, serializer):
         if isinstance(serializer, serializer_module.Serializer):
             log = json.dumps(self._log)
             serializer('_log', log)
+            log_str = self._log_str.decode('utf-8')
+            serializer('_log_str', log_str)
```

Comment:
Why is `.decode('utf-8')` needed?

Reply:
`self._log_str` holds a value as `bytearray` (see [L79](https://github.com/chainer/chainer/blob/796838f49c3fbb17ebc08a4b5152fa585a8880d6/chainer/training/extensions/log_report.py#L79)) and it needs to decode it to `serialize`.